src/apps/mailing/static/templates/transfer_ownership_unregistered_user_en.html

```diff
@@ -15,7 +15,8 @@
     </tr>
     <tr>
         <td style="padding-bottom: 1%">
-            Please visit <a href="{{url}}">here</a> to create a new account and accept it.
+            Please visit <a href="{{ link }}/{{applet_id}}?key={{ key }}&action=accept&lang={{ language }}">here</a>
```

Comment:
Pls check, that url should be passed from backend side, no need to construct url in template, moreover, this links should be directed to account creation, not to accepting. Try to understand what are you doing